UnityProject/Assets/Scripts/Tool Construction Deconstruction/MachineChassis.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MachineChassis : MonoBehaviour, IConstructionHandler
```

Comment:
Although this class is obvious it would help to add some simple summary comments above